cmd/kubeconform/validate.go

```diff
@@ -93,13 +83,11 @@ func realMain() int {
 		useStdin = true
```

Comment:
we still refer to os.Stdin here instead of cfg.stream.stdin, but here we need Stat()

Reply:
yeah, I put it away to think about the best way to make it.
I will get back to it.

Reply:
Hold on a bit - I'm not super happy with the signature of "Validate" yet, I would want it to enable the caller to process the results in a way they see fit. I need to think about this a bit more :bow:

Reply:
great :ok_hand: :ok_hand: